mycroft/audio/audioservice.py

```diff
@@ -495,6 +501,38 @@ def _list_backends(self, message):
             data[s.name] = info
         self.bus.emit(message.response(data))
 
+    def _get_track_length(self, message):
+        """
+        getting the duration of the audio in milliseconds
```

Comment:
Seems like everything for core is in seconds, whilst VLC etc may be in ms,  is that right?